dbadapter/db_adapter.go

```diff
@@ -44,14 +46,32 @@ func getMongoClient(url string, dbname string) error {
 	return errConnect
 }
 
-func ConnectMongo(url string, dbname string) {
+// Set AuthDBClient
+func setAuthDBClient(authurl string, authkeysdbname string) error {
+	var mClient, errConnect = mongoapi.NewMongoClient(authurl, authkeysdbname)
+	if mClient.Client != nil {
+		AuthDBClient = mClient
+		AuthDBClient.(*mongoapi.MongoClient).Client.Database(authkeysdbname)
+	}
+	return errConnect
+}
+
+func ConnectMongo(url string, dbname string, authurl string, authkeysdbname string) {
 	// Connect to MongoDB
+	if authurl == "" {
```

Comment:
best place to update default would be  InitConfigFactory(f string)

Reply:
Hello @thakurajayL,
The PR is updated and the  default value is set in the InitConfigFactory(f string).